apps/assisted_tagging/models.py

```diff
@@ -90,6 +90,10 @@ class PredictionStatus(models.IntegerChoices):
         DONE = 2, 'Done'
         SEND_FAILED = 3, 'Send Failed'
 
+    class DraftEntryType(models.IntegerChoices):
```

Comment:
We can do this instead. Easier to use in other places.
`DraftEntry.Type.AUTO vs DraftEntry.DraftEntryType.AUTO`
```suggestion
    class Type(models.IntegerChoices):
```